test/stdlib/Inputs/DictionaryKeyValueTypesObjC.swift

```diff
@@ -475,8 +475,7 @@ func getBridgedNSDictionaryOfRefTypesBridgedVerbatim() -> NSDictionary {
   d[TestObjCKeyTy(20)] = TestObjCValueTy(1020)
   d[TestObjCKeyTy(30)] = TestObjCValueTy(1030)
 
-  let bridged =
-    unsafeBitCast(convertDictionaryToNSDictionary(d), to: NSDictionary.self)
```

Comment:
@lorentey just to check, can you spot any higher purpose to this bit cast that I'm missing? The function already returns an `NSDictionary`.

Reply:
There is none as far as I can tell!